doc/conf.py

```diff
@@ -45,8 +45,7 @@
 templates_path = ['_templates']
 
 # The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-source_suffix = ['.rst']
+#source_suffix = {'.rst': 'restructuredtext'}
```

Comment:
Why the change? Is there no need to specify this parameter?  (= for my own education)

Reply:
Sphinx was issuing a warning
```
Converting `source_suffix = ['.rst']` to `source_suffix = {'.rst': 'restructuredtext'}`.
```
because the [preferred format for `source_suffix`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-source_suffix) has actually [changed years ago](https://www.sphinx-doc.org/en/master/changes/1.8.html#release-1-8-0-released-sep-13-2018). But it's also set to `.rst` by default, so there's no need to specify it explicitly, until we want to add something else (there are many lines in `conf.py` that show the default but are commented out). So just for consistency.